e2e/support/helpers/api/createQuestion.ts

```diff
@@ -166,6 +166,10 @@ export const question = (
           cy.intercept("POST", "/api/dataset").as("dataset");
           cy.visit(`/model/${body.id}`);
           cy.wait("@dataset"); // Wait for `result_metadata` to load
+        } else if (type === "metric") {
+          cy.intercept("POST", "/api/dataset").as("dataset");
+          cy.visit(`/metric/${body.id}`);
```

Comment:
Oh, we should use `visitMetric` here.
Also probably `visitQuestion` and `visitModel` (although this may break some existing tests).